src/main/resources/default/assets/common/core.js

```diff
@@ -43,7 +43,7 @@ sirius.addEnterListener = function(_node, listener) {
     _node.addEventListener('keydown', function(event) {
         if (event.code === 'Enter') {
```

Comment:
event.code is not supported in IE 11, we have to use keyCode as long as we support IE

Reply:
event.key can be used in the case of Enter